apps/poker-roll/src/app/graphComponent.jsx

```diff
@@ -1,75 +1,61 @@
 import React from "react";
 import { useEffect } from "react";
 import { ResponsiveLine } from '@nivo/line'
-import { useBankroll } from './queries/helpers';
+import { useGetAllBankroll } from './queries/helpers';
+import { useUser } from "@clerk/clerk-react";
 
-// const graphData = useEffect((props) => {
-//   setSeries([{
-//     id: "Bankroll",
-//     data: props.data
-//     .sort((r1,r2) => r1.timestamp - r2.timestamp)
-//     .map(bankroll => {
-//       return {
-//       x: bankroll.timestamp,
-//       y: bankroll.amount,
-//     }
-//   })}])
-//     let yValues = props.data.map(d => d.value);
-//     let minValue = yValues.reduce((v1, v2) => v1 > v2 ? v2 : v1);
-//     let maxValue = yValues.reduce((v1, v2) => v1 > v2 ? v1 : v2);
-//     setMinY(minValue - getStdDeviation(yValues));
-//     setMaxY(maxValue + getStdDeviation(yValues));}, [props.data]);
   
-const graphData = (props) => {
-    props.data.map(bankroll => {
+const graphData = (bankrolls) => {
+    return bankrolls.map(bankroll => {
     return { 
-                x: bankroll.timestamp,
+                x: new Date(bankroll.timeStamp),
                 y: bankroll.amount,
             }
         })
 };
 
-const graphPoints = () => {
-    return [ {
-        "id": "Bankroll",
-        "color": "hsl(238, 70%, 50%)",
-        "data": [ graphData() ]
-    }]
-}
 
 
 const Graph = () => {
-  const bankrollQuery = useBankroll()
+  const bankrollQuery = useGetAllBankroll();
 
     return (
+        <div>
```

Comment:
@AMess33 Went searching for this div in the elements panel and found it has a height of `0`. By default, the `ResponsiveLine` graph is set to `100%` of the height and width of its parent component. You could try tweaking your layout further up the tree to ensure there is space for this to render but the easiest thing to do here is just set a static height on this div.

<img width="422" alt="Screenshot 2023-10-18 at 9 19 04 AM" src="https://github.com/AMess33/PokerRoll/assets/20647694/b0cfa078-faa6-481b-8484-3b2f0b26b6a3">